network/testing/testing.go

```diff
@@ -86,6 +86,8 @@ type getRandomPeerDelegate func() *peer.ID
 type fetchAndSetTemporaryDialDelegate func(peer.ID, bool) bool
 type removeTemporaryDialDelegate func(peer.ID)
 
+func (m *MockNetworkingServer) TemporaryDialPeer(peerAddrInfo *peer.AddrInfo) {}
```

Comment:
Nitpick:

Can you please introduce a delegate method like we did for other `MockNetworkingServer` methods, and populate the body of this callback, like we did for other methods?

It makes it easier in the future to mock, and is consistent with the way we write mocks